how-to-guides/sagemaker/custom-docker-container/Using-Neptune-with-custom-container.ipynb

```diff
@@ -1,12 +1,15 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "abc6583e-be23-457f-9f35-d5e790aa9636",
    "metadata": {},
    "source": [
     "# Using Neptune together with Amazon SageMaker training jobs \n",
     "\n",
+    "<div class=\"alert alert-info\">You can run this part of the notebook either locally or from AWS SageMaker Notebook. It would need additional dependencies such as AWS CLI tools and Docker.</div>\n",
```

Comment:
These classes don't render in colab, so better to stick to HTML/markdown formatting

Reply:
But those notebooks won't work in Colab as they are to be run from SageMaker Notebooks that use regular Jupyter. Only the part where someone needs to build a docker image could be run from Colab, but it wouldn't be a great idea and if they don't wont to run it locally, they'd rather just run it from a SageMaker notebook all the way.